package-lock.json

```diff
@@ -144,7 +145,6 @@
         }
       ],
       "license": "MIT",
```

Comment:
[question] This file diff looks large after only install `react-helmet` package. Could you please re-check correct node version?

Reply:
I checked, we have the correct changes in the `package-lock.json` file